Plutarch/DataRepr/Internal.hs

```diff
@@ -192,7 +192,7 @@ instance
   where
   pshow' b xs = "[" <> pmatch xs go
     where
-      go :: PDataRecord ((label ':= x) : xs) s -> Term s PString
+      go :: PDataRecord ((label ':= x) ': xs) s -> Term s PString
```

Comment:
You still have a lot of these changes. Can you maybe just rebase to before you did this change?
Another option is to remove the ' entirely everywhere, but we shouldn't be adding more of them at least.

Reply:
These are the places where ticks should've been in the first place for the consistency.